rpmio/rpmkeyring.cc

```diff
@@ -61,6 +69,55 @@ rpmKeyring rpmKeyringFree(rpmKeyring keyring)
     return NULL;
 }
 
+rpmKeyringIterator rpmKeyringInitIterator(rpmKeyring keyring, int unused)
+{
+    if (!keyring || unused != 0)
+	return NULL;
+
+    keyring = rpmKeyringLink(keyring);
```

Comment:
This looks confusing - link the keyring in the iterator initialization instead.

Reply:
Done.